test/unit/widgets/buttons.test.ts

```diff
@@ -406,6 +455,34 @@ describe('personTR', () => {
   })
 })
 
+describe('renderAsDiv ', () => {
+  it('is exposed on public API', () => {
+    expect(renderAsDiv).toBe(renderAsDiv)
+  })
+
+  it('uses the image given', () => {
+    image.setAttribute('alt', 'test')
+    const options = { image: image }
+    const element = renderAsDiv(dom, obj, options)
+    expect(element.children[0].children[0].getAttribute('alt')).toEqual('test')
+  })
+  it.skip('the div is clickable if given true for the clickable option', () => {
```

Comment:
**issue** unfinished test

Reply:
I added testing-library/user-event and was able to test this now.